test/algae_dsl_aliasing_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule AlgaeDslAliasingTest.Base do
+  import Algae
+
+  alias __MODULE__
+
+  defmodule A do
+    defdata do
+      a :: String.t
```

Comment:
@icidasset Nitpick: could you add the parens after the `.t`s? That annoying modern syntax and whatnot 🤣

Reply:
@expede Sure thing 👍